ipatests/prci_definitions/gating.yaml

```diff
@@ -3,14 +3,10 @@ topologies:
     name: build
     cpu: 2
     memory: 3800
-  master_1repl: &master_1repl
```

Comment:
Tip: We have a special template for temp commits, which is easier to use. You can activate the template with a single command:

``ln -sf ipatests/prci_definitions/temp_commit.yaml .freeipa-pr-ci.yaml``

Reply:
Ok, thank you